deploy/crds/mongodb.com_mongodb_crd.yaml

```diff
@@ -50,14 +50,95 @@ spec:
               description: Members is the number of members in the replica set
               type: integer
             replicaSetHorizons:
-              description: Add this parameter and values if you need your database
```

Comment:
I'm not sure why this description changed when it was auto-generated. Should I change it back?

Reply:
Done! Note that formatting still changed a bit because of the way the comment was formatted.